contracts/DappStakingPool.sol

```diff
@@ -38,8 +41,8 @@ contract DappStakingPool is OwnableUpgradeable, ITransferPositionCallback {
     ILiquidityProtection public liquidityProtection;
     ILiquidityProtectionStore public liquidityProtectionStore;
 
-    IERC20 public dappToken;
-    IERC20 public bntToken;
+    IERC20Upgradeable public dappToken;
+    IERC20Upgradeable public bntToken;
```

Comment:
why do we use the upgradeable interface for these tokens..? Also quite sure bnt isn't actually upgradeable. Doesn't really matter, just coding conventions

Reply:
Egg-sellent question.  I think it was because hardhat was saying the contracts were not upgrade safe.  As you said, it doesn't change any business logic, full commit here https://github.com/davidberiro/bancor-liquidity-mining/commit/1575e1df07d017942a5857e72925062244eff0ad